l10n-pocket/en/footer.ftl

```diff
@@ -35,3 +36,24 @@ pocket-footer-cookie-preferences = Cookie preferences
 pocket-footer-read-it-later-brand-name = { -brand-name-read-it-later-inc }
 pocket-footer-twitter-brand-name = { -brand-name-twitter }
 pocket-footer-facebook-brand-name = { -brand-name-facebook }
+
+### Platform footer is used in some pages:
+
+## Pocket homepage https://www-dev-pocket.allizom.org
+## Add page URL https://www-dev-pocket.allizom.org/add
+## All platform pages
+## https://www-dev-pocket.allizom.org/ios
+## https://www-dev-pocket.allizom.org/android
+## https://www-dev-pocket.allizom.org/chrome
+## https://www-dev-pocket.allizom.org/safari
+## https://www-dev-pocket.allizom.org/opera
+## https://www-dev-pocket.allizom.org/edge
+
+##
```

Comment:
This signals the end of a group, but there are no strings between the start and the end here?